pkg/expr/expr_types.go

```diff
@@ -463,12 +463,12 @@ func CastToUnsignedInteger(val interface{}) (out uint64, err error) {
 }
 
 func (t *KV) Has(k string) bool {
-	_, has := t.value[k]
-	return has
+	_, is := t.value[k]
```

Comment:
why change this?

Reply:
Reverting this change, can't recall the reason but must be changed to keep all bool receivers same throughout the file 🙈